SetupDataPkg/Tools/GenCfgData_test.py

```diff
@@ -10,14 +10,30 @@
 import base64
 
 from SettingSupport.SettingsXMLLib import SettingsXMLLib        # noqa: E402
-from GenCfgData import CGenCfgData
+from GenCfgData import CGenCfgData, DICT_KEYS_KEYWORDS
 from GenNCCfgData import CGenNCCfgData
+from collections import OrderedDict
 
 
 class UefiCfgUnitTests(unittest.TestCase):
 
     # General test for loading yml file and making sure all config is there
     def test_yml_to_config(self):
+        def _search_exec(exec, item):

```

Comment:
nit: this is probably not exec anymore? I think an "exec" is supposed to be the top leaves under the root?

Reply:
Sure, I can name it something more generic.